### DISCUSSION TITLE
v2 Brainstorming

### DISCUSSION THREAD

**Discussion by imreallyliam** [dahliaOS/terminal#7] [General]
Related to PR #6:

- [ ] Use pty (or similar) for cross-platform access.
- [ ] Use xterm instead of the current implementation (incompatible with xterm standards)

Currently, the xterm dependency is having issues working with pty, a future version is making strides towards a fix.

**Comment by carbonatedcaffeine** [comment]
Have you seen this other flutter terminal implementation?

**Reply by imreallyliam** [reply]
If you mean the v1 terminal, yes. Discussed with @nmcain and we thought that the terminal was overall due for a backend rewrite. I'm aiming to use components that already exist as to not increase our efforts. This is why I'm _trying_ to get xterm to cooperate, not to mention it's up to date on xterm standards.
